constants/locale.go

```diff
@@ -2,508 +2,3360 @@ package constants
+	&Locale{
+		Locale:    "my_MM",
+		Title:     "Burmese language locale for Myanmar",
+		Language:  "Burmese",
+		Territory: "Myanmar",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "nan_TW",
+		Title:     "Min Nan Chinese locale for the Republic of China",
+		Language:  "Min Nan Chinese",
+		Territory: "Republic of China",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "nan_TW@latin",
+		Title:     "Minnan language locale for Taiwan",
+		Language:  "Minnan",
+		Territory: "Taiwan",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "nb_NO",
+		Title:     "Norwegian (Bokmal) locale for Norway",
+		Language:  "Norwegian, Bokm\xe5l",
+		Territory: "Norway",
+		Codeset:   "ISO-8859-1",
+	},
+	&Locale{
+		Locale:    "nb_NO.utf8",
+		Title:     "Norwegian (Bokmal) locale for Norway",
+		Language:  "Norwegian, Bokmål",
+		Territory: "Norway",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "nds_DE",
+		Title:     "Low(lands) Saxon Language Locale for Germany",
+		Language:  "nds",
+		Territory: "DE",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "nds_NL",
+		Title:     "Low(lands) Saxon Language Locale for the Netherlands",
+		Language:  "nds",
+		Territory: "NL",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "ne_NP",
+		Title:     "Nepali language locale for Nepal",
+		Language:  "Nepali",
+		Territory: "Nepal",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "nhn_MX",
+		Title:     "Central Nahuatl for Mexico",
+		Language:  "Central Nahuatl",
+		Territory: "Mexico",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "niu_NU",
+		Title:     "Niuean (Vagahau Niue) locale for Niue",
+		Language:  "Vagahau Niue (Niuean)",
+		Territory: "Niue",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "niu_NZ",
+		Title:     "Niuean (Vagahau Niue) locale for New Zealand",
+		Language:  "Vagahau Niue (Niuean)",
+		Territory: "New Zealand",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "nl_AW",
+		Title:     "Dutch language locale for Aruba",
+		Language:  "Dutch",
+		Territory: "Aruba",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "nl_BE",
+		Title:     "Dutch locale for Belgium",
+		Language:  "Dutch",
+		Territory: "Belgium",
+		Codeset:   "ISO-8859-1",
+	},
+	&Locale{
+		Locale:    "nl_BE@euro",
+		Title:     "Dutch locale for Belgium with Euro",
+		Language:  "Dutch",
+		Territory: "Belgium",
+		Codeset:   "ISO-8859-15",
+	},
+	&Locale{
+		Locale:    "nl_BE.utf8",
+		Title:     "Dutch locale for Belgium",
+		Language:  "Dutch",
+		Territory: "Belgium",
+		Codeset:   "UTF-8",
+	},
+	&Locale{
+		Locale:    "nl_NL",
+		Title:     "Dutch locale for the Netherlands",
+		Language:  "Dutch",
+		Territory: "Netherlands",
+		Codeset:   "ISO-8859-1",
+	},
+	&Locale{
+		Locale:    "nl_NL@euro",
+		Title:     "Dutch locale for the Netherlands with Euro",
+		Language:  "Dutch",
+		Territory: "Netherlands",
+		Codeset:   "ISO-8859-15",
+	},
+	&Locale{
+		Locale:    "nl_NL.utf8",
+		Title:     "Dutch locale for the
```

Comment:
__[golint]__
* [golint] exported method LocaleList.Strings should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->